src/ss_fact.jl

```diff
@@ -42,32 +87,35 @@ function sspdmp_inner!(Ξ, G, G2, ∇ϕ, t, x, θ, Q, c, b, t_old, f, θf, (acc,
             t_old[i] = t[i]
             f[i] = false
             Q[i] = t[i] - log(rand())/κ
```

Comment:
We need to make kappa a vector
```suggestion
            Q[i] = t[i] - log(rand())/κ[i]
```

Reply:
:+1: